kong/plugins/jwt/migrations/postgres.lua

```diff
@@ -104,4 +104,22 @@ return {
     end,
     down = function(_, _, dao) end  -- not implemented
   },
+  {
+    name = "2018-05-28-151700_jwt_inject_claims_default",
+    up = function(_, _, dao)
+      for ok, config, update in plugin_config_iterator(dao, "jwt") do
+        if not ok then
+          return config
+        end
+        if config.inject_claims == nil then
+          config.inject_claims = {}
```

Comment:
It would result in faster and safer migrations if this was included in the above migration instead. We should also rename it to something like `jwt_max_exp_and_inject_claims`.